### DISCUSSION TITLE
Encouraging people to share their datasets

### DISCUSSION THREAD

**Discussion by lolcharacters** [civitai/civitai#213] [Ideas]
We should encourage people to share datasets by changing the design of the website.

Pros
- In the future, we'll eventually get better base models or innovations in AI research that will improve the quality of AI art considerably. This will require retraining of every TI/LoRA which will be much harder if people don't share their dataset.
- Sharing datasets allow others to improve on the existing upload.
- It also allows others to contribute to the dataset when new images are created.

Cons
- Authors feel less ownership / less incentive for leaderboard
- Will lead to make more takedowns? Artists can see what exactly was used.

People very rarely upload their datasets to huggingface either. I'm not sure how we encourage people to share datasets. Forcing people to share seems a bit too much. Opening this up for discussion/ideas

**Comment by JustMaier** [comment]
I completely agree with this. It's something I've discussed with the community a few times over the past month or so. Like you said, the state of the art is moving fast, but something that is always going to be needed is quality training data. If we could encourage more people to include training data, it would make it possible for people to easily "upgrade" their model, LoRA, etc. to whatever the new standard may be.

Regarding the Cons:
- I think we could find a way to incentivize the sharing of training data to account for any concern from model creators. Perhaps we even give extra points to them in the leaderboard calculation.
- Takedowns are indeed a concern. Specifically, I believe the hosting of copyrighted work would be problematic. To address that, I believe we could consider instead just storing captions and links to the original images. However, that seems fragile and likely difficult for the individuals preparing the datasets, especially since some images may or may not actually come from the internet. I'm not really sure how to solve this one. Anyone reading this have any ideas?
